pom.xml

```diff
@@ -33,8 +33,6 @@
     </modules>
 
     <properties>
-        <java.version>11</java.version>
```

Comment:
Does this need to go?

Reply:
It was being used as a property to set the version in the compiler plugin but this is now being set directly and it's not  used anywhere else. `java.version` is not a built in maven property, it's a [property used by Spring Boot](https://stackoverflow.com/a/38883073). We are not using that so there no need for it to exist.

Reply:
Fair enough, was just thinking even though it's used once it might be a good way to highlight the version